test/tools/isoltest.cpp

```diff
@@ -480,9 +481,20 @@ int main(int argc, char const *argv[])
 
 		return global_stats ? 0 : 1;
 	}
-	catch (std::exception const& _exception)
+	catch (boost::program_options::error const& exception)
 	{
-		cerr << _exception.what() << endl;
+		cerr << exception.what() << endl;
+		return 1;
+	}
+	catch (std::runtime_error const& exception)
+	{
+		cerr << exception.what() << endl;
+		return 1;
+	}
+	catch (...)
+	{
+		cerr << "Uncaught exception" << endl;
```

Comment:
```suggestion
		cerr << "Uncaught exception." << endl;
```
Actually. You just caught it. So probably you want to write:
```suggestion
		cerr << "Unhandled exception caught." << endl;
```

Reply:
Well, it really means that it was not caught until now but you have a point :) For the user seeing this message this might not be clear. I changed the message to your second version.